src/plural.js

```diff
@@ -110,5 +110,18 @@ angular.module("gettext").factory("gettextPlurals", function () {
             default: // Everything else
                 return n != 1 ? 1 : 0;
         }
+    };
+
+    /**
+     * Method extracts iso639-2 language code from code with locale e.g. pl_PL, en_US, etc.
+     * If it's provided with standalone iso639-2 language code it simply returns it.
+     * @param {String} langCode
+     * @returns {String} iso639-2 language Code
+     */
+    function getLanguageCode(langCode) {
+        if( langCode === 'pt_BR' ) {
+            return 'pt_BR';
+        }
+        return langCode.split('_').shift();
```

Comment:
You can reduce the cost of this method by adding a small cache within `gettextPlurals`:

```js
var languageCodes = {
    'pt_BR': 'pt_BR',
};
```

Then inside `getLanguageCode`:

```js
if (!languageCodes[langCode]) {
     languageCodes[langCode] = langCode.split('_').shift();
}
return languageCodes[langCode];
```

That should bring the extra cost of this down to almost zero.